File: lista_Logica_v2/ex3.py
# 3) Escreva um algoritmo que leia n valores numéricos (quantos o usuário quiser digitar) e encontre o maior, o menor deles e a média aritmética. Mostre o resultado.

continua = 1
cont = 0
soma = 0
while (continua != 0):
    num = float(input("Digite um número: "))
    if (cont == 0): # Para colocar o primeiro valor como maior e menor, ou seja, o referencial inicial
        maior = num
        menor = num
    if num > maior:
        maior = num
    if num < menor:
        menor = num
    soma += num
    cont += 1
    continua = int(input("Digite 0 para sair ou 1 para continuar: "))

print(f"Maior número: {maior} | Menor número: {menor} | Média: {(soma / cont):.2f}")

File: lista_Logica_v2/emC/ex3.c
#include <stdio.h>

void main() {
    int continua = 1;
    int cont = 0;
    float soma = 0;
    float num, maior, menor;
    
    while (continua != 0) {
        printf("Digite um numero: ");
        scanf("%f", &num);
        
        if (cont == 0) { // Para colocar o primeiro valor como maior e menor
            maior = num;
            menor = num;
        }
        
        if (num > maior) {
            maior = num;
        }
        
        if (num < menor) {
            menor = num;
        }
        
        soma = soma + num;
        cont = cont + 1;
        
        printf("Digite 0 para sair ou 1 para continuar: ");
        scanf("%d", &continua);
    }
    printf("Maior numero: %f | Menor numero: %f | Media: %f\n", maior, menor, (soma/cont));
}